### DISCUSSION TITLE
To be or to be not a validation library

### DISCUSSION THREAD

**Discussion by IDrumsey** [pydantic/pydantic#7926] [General]
The documentation is a little contradictory about whether or not Pydantic is a validation library. On the [home page](https://docs.pydantic.dev/latest/) of the docs, it says
> Pydantic is the most widely used data validation library for Python.

However on the [models page](https://docs.pydantic.dev/latest/concepts/models/), it has the info box stating, 
> Pydantic is primarily a parsing and transformation library, not a validation library.

Also in the announcement found [here](https://pydantic.dev/announcement/), Samuel Colvin states,
> I've watched with fascination as Pydantic has grown to become the most widely used Python data validation library

Unless I'm missing something, these are conflicting statements about the nature of the Pydantic library.

**Comment by sydney-runkle** [comment]
@IDrumsey,

Good question. We have an open issue regarding this confusion here: https://github.com/pydantic/pydantic/issues/3884

Given that it's caused confusion for lots of people, I can prioritize making these changes to the docs in the next week or so 👍

**Reply by sydney-runkle** [reply]
@IDrumsey, update coming here:

https://github.com/pydantic/pydantic/pull/8080
